serverless/test/env.spec.ts

```diff
@@ -249,6 +260,7 @@ describe("validateParameters", () => {
       enableDDLogs: true,
       enableEnhancedMetrics: true,
       extensionLayerVersion: 6,
+      captureLambdaPayload: false,
```

Comment:
While trivial, it might be a good idea to add a test case for captureLambdaPayload:true